mon-pix/app/models/campaign-participation-overview.js

```diff
@@ -11,13 +11,14 @@ export default class CampaignParticipationOverviews extends Model {
   @attr('string') campaignCode;
   @attr('string') campaignTitle;
   @attr('date') campaignArchivedAt;
+  @attr('boolean') campaignIsForAbsoluteNovice;
   @attr('number') masteryPercentage;
   @attr('number') totalStagesCount;
   @attr('number') validatedStagesCount;
 
   get status() {
     if (this.campaignArchivedAt) return 'ARCHIVED';
-    else if (this.isShared) return 'ENDED';
+    else if (this.isShared || (this.assessmentState === 'completed' && this.campaignIsForAbsoluteNovice)) return 'ENDED';
```

Comment:
je pense qu'il faudrait changer côté back, parce que le tri est mauvais aussi : on le tri comme s'il était "to share" alors qu'on affiche "terminé"